_assets/icons/about.svg

```diff
@@ -0,0 +1,8 @@
+<svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<defs>
+<symbol id="icon-about" viewBox="0 0 16 16">
```

Comment:
Tested using xlink:href without referencing the symbol ID, SVG wouldn't display. Likewise tested using href and this also failed to display the icon.